Rakefile

```diff
@@ -27,6 +27,16 @@ RSpec::Core::RakeTask.new(:acceptance) do |t|
     t.pattern = 'spec/acceptance'
 end
 
+desc "Run integration tests"
+RSpec::Core::RakeTask.new(:integration) do |t|
+  t.pattern = 'spec/acceptance/integration'
+  ENV['BEAKER_setfile'] ||= 'spec/acceptance/beaker/nodesets/rhel7.yaml'
+  ENV['SPEC_FORGE'] ||= 'https://api-forge-aio01-qatest.puppetlabs.com/'
```

Comment:
This should be a service name, not the actually host name (modules-staging.puppetlabs.com)